smithy-aws-protocol-tests/model/restJson1/main.smithy

```diff
@@ -86,5 +86,8 @@ service RestJson {
         // @endpoint and @hostLabel trait tests
         EndpointOperation,
         EndpointWithHostLabelOperation,
+
+        // custom endpoints with paths
+        PathOperation,
```

Comment:
`PathOperation` reads like an operation with path parameters. Maybe EndpointWithPathOperation? Or if that implies use of @endpoint trait like the couple of tests above, maybe HostWithPathOperation?